solutions/fscloud/README.md

```diff
@@ -0,0 +1,9 @@
+# IBM Elastic search fscloud on IBM Cloud
+
+This architecture creates an IBM Elastic search fscloud instance in IBM Cloud  environment and supports provisioning the following resources:
+
+- A resource group, if one is not passed in.
+- An IBM Elastic search fscloud instance on IBM Cloud
+- A IBM Elastic search database
```

Comment:
Lets name it as - IBM Elastic Search on IBM Cloud

As such there is nothing we call it `fscloud instance`. Please check the Cloud docs for the official naming convention.

Reply:
Lets not use fscloud anywhere.

Reply:
For diagram, refer here - https://github.com/terraform-ibm-modules/terraform-ibm-event-streams/pull/181/files

We need to make type of. Check that what elastic search module provides and accordingly make changes.

Reply:
made the changes